src/sshagent/KeeAgentSettings.cpp

```diff
@@ -50,6 +50,15 @@ bool KeeAgentSettings::isDefault() const
     return (*this == defaultSettings);
 }
 
+/**
+ * Reset this instance to default settings
+ */
+void KeeAgentSettings::reset()
+{
+    KeeAgentSettings defaultSettings;
+    *this = defaultSettings;
```

Comment:
I am not a fan of this method of resetting.

Reply:
Did exactly this now.